lib/fog/aws/requests/elasticache/create_cache_cluster.rb

```diff
@@ -19,7 +19,7 @@ class Real
         #   * :engine_version <~String> - The Cluster's caching software version
         #   * :notification_topic_arn <~String> - Amazon SNS Resource Name
         #   * :port <~Integer> - The memcached port number
-        #   * :preferred_availablility_zone <~String>
+        #   * :preferred_availabilility_zone <~String>
```

Comment:
This correction looks off, I think it needs one fewer `il` in the middle? That should match with the other change below and the related one in the other file above.